server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_11_alter_regulation_riskdetect_rule.sql

```diff
@@ -0,0 +1,5 @@
+--
+-- Alter column `value_json` size from varchar(2048) to MEDIUMTEXT in `regulation_riskdetect_rule` table
+--
+ALTER TABLE `regulation_riskdetect_rule` CHANGE COLUMN `value_json` `value_json_old` VARCHAR(2048);
+ALTER TABLE `regulation_riskdetect_rule` ADD COLUMN `value_json` MEDIUMTEXT DEFAULT NULL;
```

Comment:
how to process the data that contained by `value_json_old`?

Reply:
update `value_json_old` to `value_json`